server/routes/viewingLicences/handlers/viewLicence.ts

```diff
@@ -10,8 +10,15 @@ export default class ViewAndPrintLicenceRoutes {
 
   GET = async (req: Request, res: Response): Promise<void> => {
     const { licence, user } = res.locals
-
     let warningMessage
+
+    const releaseDateString = licence.actualReleaseDate || licence.conditionalReleaseDate
+    const releaseDate = parse(releaseDateString, 'dd/MM/yyyy', new Date())
+    const isInHardStopWindow = isWithinInterval(startOfDay(new Date()), {
+      start: sub(releaseDate, { days: 2 }),
+      end: releaseDate,
+    })
```

Comment:
This can be removed/replaced when the hard-stop logic is fully implemented on the licence representation